fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -548,6 +549,13 @@ public void execute(TUniqueId queryId) throws Exception {
                                     olapScanNode.getCardinality(),
                                     context.getQualifiedUser());
                         }
+
+                        // set table names in fe.audit.log
```

Comment:
Few questions:
1. How to handle duplicate table name, there may be 2 scan node will same table.
2. How to handle same table name from different database?

Reply:
Thanks for your reply , It's a good suggestion for me.